libexec/nodenv-versions

```diff
@@ -97,13 +97,17 @@ while read -r version; do
 done < <(
   {
     shopt -s nullglob
-    for path in "$versions_dir"/*; do
+    for path in "$versions_dir"/* "$versions_dir"/lts/*; do
       if [ -d "$path" ]; then
+        if [[ $path =~ /lts(/bin)?$ && ! -L $path ]]; then
```

Comment:
The theoretically allows something like `ln -s 12 lts/bin` which would be weird but I don't really see a reason to necessarily prevent it and thereby make this condition more complicated. Happy to adjust thought if you disagree.

Reply:
This won't quite work because if someone adds a node named `lts`, it would still list all the other subdirs of that version as versions.

What we need to do here, (which I'm working on locally) is to ignore any version that itself doesn't contain a bin directory. This is a more significant change, because it will cause existing users to stop listing directories under their nodenv versions directory that may already exist with a bin subdir. But I would consider those to already be errors (since without a bin/, there's nothing from that version for nodenv's shims to delegate to, ergo its a noop node).